src/main/java/org/alliancegenome/curation_api/services/DiseaseAnnotationFmsService.java

```diff
@@ -43,6 +42,12 @@
     @Inject ExperimentalConditionDAO experimentalConditionDAO;
     @Inject BiologicalEntityDAO biologicalEntityDAO;
     @Inject GeneDAO geneDAO;
+    @Inject VocabularyTermDAO vocabularyTermDAO;
+    
+    private String CONDITION_RELATION_TYPE_VOCABULARY = "Condition relation types";
+    private String GENE_DISEASE_RELATION_VOCABULARY = "Gene disease relations";
+    private String ALLELE_DISEASE_RELATION_VOCABULARY = "Allele disease relations";
+    private String AGM_DISEASE_RELATION_VOCABULARY = "AGM disease relations";
```

Comment:
These would probably be better pulled out into a `Constants` class. And typically they are defined as:

`private static final String CONSTANT = "CONSTANT";`

Reply:
Moved to VocabularyConstants class